### PR TITLE
Use webpack for the build

### PR DESCRIPTION
**Pull request by gabro**

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+
+# End of https://www.gitignore.io/api/node
+
+/out.js
```
Comment (gabro) [inline client/.gitignore:67]: I used a generated gitignore + out.js

File: client/package.json [hunk 2]
```diff
@@ -9,15 +9,13 @@
...
+    "build": "webpack"
   },
   "devDependencies": {
-    "browserify": "latest",
-    "babelify": "latest",
-    "babel-plugin-transform-react-jsx": "latest"
-  },
-  "browserify": {
-   "transform": ["babelify"]
+    "babel-core": "^6.26.0",
```
Comment (gabro) [inline client/package.json:16]: `babel-loader` lists `babel-core` as its peerDependency, so we need to explicitly depend on it.

File: client/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  entry: './main.js',
+  output: {
+    filename: 'out.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
```
Comment (gabro) [inline client/webpack.config.js:10]: this uses `.babelrc` by default. Alternatively one could remove the `.babelrc` and just inline the babel config here.

**Comment by andreaTP** [comment]
Thanks a lot!
Extremely helpful!

**Pull request merged**